### ISSUE TITLE
Issues when you trying to run

### ISSUE THREAD

**Issue by anacondaq** [AndhikaWB/BuildToolsInspector#1]
2 errors on the screenshot:

![](https://i.imgur.com/8nfWShQ.png)

Error 1:

> FileNotFoundError: [Errno 2] No such file or directory: 'd:\\Downloads\\browser\\BuildToolsInspector-master\\data\\channel_info.json'

I.e. probably not downloaded the file and not placed to this folder.
I've checked src, and found out that nothing downloaded at all when script running. Try yourself.
https://aka.ms/vs/17/release/channel > this is not downloaded at all.

**Comment by anacondaq** [comment]
Another problem if you add the channel file above (and rename it to channel_info.json)

![](https://i.imgur.com/5N0WBRM.png)

```
---> [50](vscode-notebook-cell:/d%3A/Downloads/browser/BuildToolsInspector-master/main.ipynb#X55sZmlsZQ%3D%3D?line=49) with open(f'{install_path}\\vs_env_setup.bat', 'w') as f:
```

FileNotFoundError: [Errno 2] No such file or directory: 'C:\\BuildTools\\vs_env_setup.bat'

**Comment by AndhikaWB** [comment]
The first one (`channel_info_path` error) is now fixed on the latest commit ([b487cda](https://github.com/AndhikaWB/BuildToolsInspector/commit/b487cda5f8f1048efd901fd6f1d4dac78d5347a9)).

The second one (`C:\BuildTools\vs_env_setup.bat` error) is kinda expected if you set `vs_package_extract` to false (the default behavior). In case you're not aware, you can change some of the settings at the beginning of the script (e.g. `vs_package_extract`, `vs_package_win_sdk`) based on your needs. With the latest commit, I added a confirmation input in case the user still want to generate setup files even if nothing is extracted/downloaded.

@anacondaq Please try the latest commit and feel free to report back to me if these problems still exist.

**Comment by AndhikaWB** [comment]
Closing this issue as I think it has been resolved? Feel free to reopen if the problems still exist.
